src/components/ProgramRecordsList.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+
+import { ChevronLeft, Info } from '@edx/paragon/icons';
+import { Alert, Hyperlink } from '@edx/paragon';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
+import { getConfig } from '@edx/frontend-platform/config';
+import { logError } from '@edx/frontend-platform/logging';
+import _ from 'lodash';
+import getProgramRecords from './data/service';
+
+const ProgramRecordsList = () => {
+  const renderProfile = () => {
+    const { username } = getAuthenticatedUser();
+    return (
+      <Hyperlink
+        destination={`${getConfig().LMS_BASE_URL}/u/${username}`}
+        className="mt-2 mb-4 d-inline-block muted-link pl-3 pr-3"
+      >
+        <ChevronLeft className="mb-1" />
+        <FormattedMessage
```

Comment:
I’m not remembering off the top of my head how translations work with this `FormattedMessage` component. Is there supposed to be a file or directory somewhere else that corresponds to this `id` value? Or is all that automagically generated when you use this nifty component?

Reply:
There will be some files that need to be generated for the translation jobs to work correctly. I am going to take that work on in a separate PR. I wanted Maya to focus on the table component and I will take some of the less glorious work (like generating the empty files for translations and setting up the Jenkins/Transifex pieces).